utils/src/test/scala/za/co/absa/enceladus/utils/validation/field/ScalarFieldValidatorSuite.scala

```diff
@@ -0,0 +1,36 @@
+package za.co.absa.enceladus.utils.validation.field
+
+import org.apache.spark.sql.types.{MetadataBuilder, StringType, StructField}
+import org.scalatest.FunSuite
+import za.co.absa.enceladus.utils.schema.MetadataKeys
+import za.co.absa.enceladus.utils.types.{Defaults, GlobalDefaults, TypedStructField}
+import za.co.absa.enceladus.utils.validation.ValidationError
+
+class ScalarFieldValidatorSuite extends FunSuite {
```

Comment:
This seems unrelated to the PR, but I'm glad we have more tests :D

Reply:
Wonder how it got here. I wasn't explicitly including it, for sure. Some effect of multiple rebases, maybe.
I will check if it's valid and based on that decide on the test suite fate :-)

Reply:
Yup, a valid test. It was just missing the Copyright header (similarly a few other tests in the folder). Keeping it.
Its appearance is still a mystery though :-D

Reply:
All good then.